### PR TITLE
Issue 1835: Tone down log levels, for 'normal' errors.

### LINKED ISSUE #1835: "Normal exceptions" don't need to be logged at error level.
**Problem description**
There are many exceptions on the server that are 'normal' from a client point of view. Like attempting to read from a sealed segment. Or discovering you are connected to the wrong host. These don't need to be logged at error level with full stack traces.

**Problem location**
PravegaRequestHandler.

**Suggestions for an improvement**
Change these into info level logs without stack traces.

### PR DESCRIPTION
**Pull request by tkaitchuck**
**Change log description**
Reduces log spam and increases usefulness of messages logged.

**Purpose of the change**
Fixes #1835

**What the code does**
Changes log lines.

**How to verify it**
There should be no impact.

### CODE REVIEW

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java [hunk 1]
```diff
@@ -357,24 +357,28 @@ private void handleException(long requestId, String segment, String operation, T
             u = u.getCause();
         }
 
-        log.error("Error (Segment = '{}', Operation = '{}')", segment, operation, u);
         if (u instanceof StreamSegmentExistsException) {
+            log.info("Segment '{}' already exists and cannot perfrom operation '{}'", segment, operation);
```
Comment (fpj) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java]: Typo in `perfrom`.

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java [hunk 2]
```diff
@@ -357,24 +357,28 @@ private void handleException(long requestId, String segment, String operation, T
...
             connection.send(new SegmentAlreadyExists(requestId, segment));
         } else if (u instanceof StreamSegmentNotExistsException) {
+            log.info("Segment '{}' does not exist and cannot perfrom operation '{}'", segment, operation);
```
Comment (fpj) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java]: Typo in `perfrom`.

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java [hunk 3]
```diff
@@ -357,24 +357,28 @@ private void handleException(long requestId, String segment, String operation, T
...
             connection.send(new NoSuchSegment(requestId, segment));
         } else if (u instanceof StreamSegmentSealedException) {
+            log.info("Segment '{}' is sealed and cannot perfrom operation '{}'", segment, operation);
```
Comment (fpj) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java]: Typo in `perfrom`.

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java [hunk 4]
```diff
@@ -357,24 +357,28 @@ private void handleException(long requestId, String segment, String operation, T
...
         } else if (u instanceof WrongHostException) {
+            log.info("Wrong host. Segment = '{}' is not owned. Operation = '{}')", segment, operation);
             WrongHostException wrongHost = (WrongHostException) u;
             connection.send(new WrongHost(requestId, wrongHost.getStreamSegmentName(), wrongHost.getCorrectHost()));
         }  else if (u instanceof ContainerNotFoundException) {
+            log.info("Wrong host. Segment = '{}' is not owned. Operation = '{}')", segment, operation);
             connection.send(new WrongHost(requestId, segment, ""));
         } else if (u instanceof CancellationException) {
-            log.info("Closing connection due to: ", u.getMessage());
+            log.info("Closing connection {} while perfroming {} due to {} ", connection, operation, u.getMessage());
```
Comment (fpj) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java]: Typo in `perfrom`.

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java [hunk 5]
```diff
@@ -269,24 +269,30 @@ private void handleException(UUID writerId, long requestId, String segment, Stri
             u = u.getCause();
         }
 
-        log.error("Error (Segment = '{}', Operation = 'append')", segment, u);
         if (u instanceof StreamSegmentExistsException) {
+            log.info("Segment '{}' already exists and {} cannot perfrom operation '{}'", segment, writerId, doingWhat);
```
Comment (andreipaduroiu) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java]: and here

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java [hunk 6]
```diff
@@ -269,24 +269,30 @@ private void handleException(UUID writerId, long requestId, String segment, Stri
...
             connection.send(new SegmentAlreadyExists(requestId, segment));
         } else if (u instanceof StreamSegmentNotExistsException) {
+            log.info("Segment '{}' does not exist and {} cannot perfrom operation '{}'", segment, writerId, doingWhat);
             connection.send(new NoSuchSegment(requestId, segment));
         } else if (u instanceof StreamSegmentSealedException) {
+            log.info("Segment '{}' does not exist and {} cannot perfrom operation '{}'", segment, writerId, doingWhat);
             connection.send(new SegmentIsSealed(requestId, segment));
+            log.info("Segment '{}' is sealed and {} cannot perfrom operation '{}'", segment, writerId, doingWhat);
```
Comment (andreipaduroiu) [inline segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java]: and here

File: segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java [hunk 7]
```diff
@@ -269,24 +269,30 @@ private void handleException(UUID writerId, long requestId, String segment, Stri
             u = u.getCause();
         }
 
-        log.error("Error (Segment = '{}', Operation = 'append')", segment, u);
```
[the remark this answers is not in the document]
Comment (tkaitchuck) [reply segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java]: I went through them and upgraded all the ones that fit the definition to warn

**Pull request merged**